books/architecture/src/resources/fs/paths.md

```diff
@@ -0,0 +1,87 @@
+# Index of PATHs
```

Comment:
Do we only save database files currently?

Reply:
we also save a `p2p_state` file, which I should add to `helper`